NitroxLauncher/LauncherLogic.cs

```diff
@@ -88,12 +88,52 @@ public async void CheckNitroxVersion()
         [Conditional("RELEASE")]
         public async void ConfigureFirewall()
         {
-            Task task = Task.Run(() => WindowsHelper.CheckFirewallRules());
-            await task;
+            Log.Info($"Using {Environment.OSVersion}");
 
-            if (task.Exception != null)
+            if (Environment.OSVersion.Platform != PlatformID.Win32NT)
             {
-                MessageBox.Show($"An error occurred configuring the firewall: {task.Exception}");
+                return;
+            }
+
+            /**
+                This feature won't work in older windows version and will crash the launcher instantly
+                Windows Vista : 6.0
+                Windows 7 : 6.1
+                Windows 8 : 6.2, Windows 8.1 : 6.3
+            **/
```

Comment:
To me this is slightly unclear. Is that it only works on these operating systems or it doesn't work on these? I think maybe a small tweak to the text to explain further, maybe to say it wont work on the below, which is what I believe the if statement below is checking. 

Does this mean we only support windows 10 and upwards now?

Reply:
Seems pretty clear to me, we're checking the windows major version here. If it's 6.x then it's a return

Reply:
Right now you can't open the launcher on W7 and older due to this feature